crates/fuel-core/src/graphql_api/storage/contracts.rs

```diff
@@ -31,6 +40,24 @@ impl TableWithBlueprint for ContractsInfo {
     }
 }
 
+impl AsTable<ContractsInfo> for StateConfig {
+    fn as_table(&self) -> Vec<TableEntry<ContractsInfo>> {
+        self.contracts
+            .iter()
+            .map(|contracts_config| TableEntry {
+                key: contracts_config.contract_id,
+                value: ContractsInfoType::V1(Salt::zeroed().into()),
+            })
+            .collect()
+    }
+}
+
+impl AddTable<ContractsInfo> for StateConfig {
+    fn add(&mut self, _entries: Vec<TableEntry<ContractsInfo>>) {
+        // Do not include these
+    }
+}
```

Comment:
All other `AddTable` implemetnations are for `StateConfigBuilder` and not `StateConfig`, I suppose we'd want to do the same here

```suggestion
impl AddTable<ContractsInfo> for StateConfigBuilder {
    fn add(&mut self, _entries: Vec<TableEntry<ContractsInfo>>) {
        // Do not include these
    }
}
```

Reply:
Also I don't see where this is used. What do we need this implementation for?

Reply:
eh, was left over from my testing, removed in 717ba42ceb

Reply:
Thanks!

Reply:
I took the liberty to remove this from the description as well 
![Uploading image.png…]()